package-lock.json

```diff
@@ -46395,7 +46395,7 @@
         "react": "^17.0.2",
         "react-dom": "^17.0.2",
         "reflux": "^0.4.1",
-        "reflux-state-mixin": "git+ssh://git@github.com/mongodb-js/reflux-state-mixin.git#e050454cb3be029c3e7fd2ee6a08111e4d15161f",
```

Comment:
These are not related to my changes, but will this update cause any issue? Seems like we are pinning it down to a particular commit?

Reply:
Outdated, its gone now